pysat/tests/test_instrument.py

```diff
@@ -345,7 +346,7 @@ def test_basic_repr(self):
         # No loaded data
         assert output.find('No loaded data') > 0
         assert output.find('Number of variables:') < 0
-        assert output.find('dummy') < 0
+        assert output.find('uts') < 0
```

Comment:
'dummy' not present in 2d data.  switching to uts for the variable check as this is present in all instruments.